evap/results/templates/results_course_detail.html

```diff
@@ -48,12 +48,12 @@
             <table class="table">
                 <thead>
                     <tr>
-                        <th class="col-sm-1">{% trans "Degree" %}</th>
-                        <th class="col-sm-2">{% trans "Type" %}</th>
-                        <th class="col-sm-3">{% trans "Responsible" %}</th>
-                        <th class="col-sm-2">{% trans "Average Grade" %}</th>
-                        <th class="col-sm-2">{% trans "Standard Deviation" %}</th>
-                        <th class="col-sm-2">{% if course.is_single_result|is_false %}{% trans "Voters" %}{% endif %}</th>
+                        <th class="col-xs-1">{% trans "Degree" %}</th>
+                        <th class="col-xs-2">{% trans "Type" %}</th>
+                        <th class="col-xs-3">{% trans "Responsible" %}</th>
+                        <th class="col-xs-2">{% trans "Average Grade" %}</th>
+                        <th class="col-xs-2">{% trans "Standard Deviation" %}</th>
+                        <th class="col-xs-2">{% if course.is_single_result|is_false %}{% trans "Voters" %}{% endif %}</th>
```

Comment:
using `xs` this also works for printing. it seems like printed pages are dealt with as xs-devices (see bootstrap docu for details, it doesn't work without in our case).
we could think about refactoring these on all pages.